docs/administration-guide/databases/mongodb.md

```diff
@@ -0,0 +1,55 @@
+# Working with MongoDB in Metabase
+
+
+This guide covers:
+
+ - What's new in `0.32.9`
+ - General connectivity concerns.
+ - Connecting to a MongoDB Atlas cluster.
+
+## What's new in `0.32.9`
+
+ - In `0.32.9`, Metabase adds support/fixes for certain MongoDB connection configurations, such as:
+   1. When connecting to an Atlas cluster, connect using `DNS SRV`, which is the recommended and default method for newer clusters.  
+
+
+## General connectivity concerns
+ 
+ - Have you checked your cluster host whitelist?  When testing a connection but seeing failure, have you tried setting the IP whitelist to `0.0.0.0/0`?  This allows connections from any IP addresses. If you know the IP address(es) or CIDR block of clients, use that instead.
+ 
+ - Are you using self-signed certificates?  While Metabase doesn't support providing a self-signed cert through the Metabase UI, you can still use the command line to configure this.  Just copy the existing store and add the self-signed cert to it like this:
+    Copy existing store and add self-signed cert to it:
```

Comment:
> Copy existing store and add self-signed cert to it:

I don't think you need this line anymore after the changes you made to the preceding sentence. Seems redundant.